packages/London-Boroughs/README.md

```diff
@@ -0,0 +1,50 @@
+# London, Boroughs
+
+[![License: Unlicense](https://img.shields.io/badge/license-Unlicense-blue.svg)](http://unlicense.org/)
+
+Contains all the 33 boroughs of London:
+* Barking & Dagenham
+* Barnet
+* Bexley
+* Brent
+* Bromley
+* Camden Town
+* City (also possible to combine this with Southwark)
+* Croydon
+* Ealing
+* Enfield
+* Greenwich
+* Hackney
+* Hammersmith & Fulham
+* Haringey
+* Harrow
+* Havering
+* Hillingdon
+* Hounslow
+* Islington
+* Kensington & Chelsea
+* Kingston upon Thames
+* Lambeth
+* Lewisham
+* Merton
+* Newham
+* Redbridge
+* Richmond upon Thames
+* Southwark
+* Sutton
+* Tower Hamlets
+* Walthan Forest
+* Wansworth
+* Westminster
+
+
+## Installation
+
+### npm
+
+`npm install --save @svg-maps/London-Boroughs`
+
+### yarn
+
+`yarn add @svg-maps/London-Boroughs`
+
```

Comment:
Could you add the credits section describing your modifications please? You can look at the other maps' README as an example